exercises/flatten-array/flatten_array_test.exs

```diff
@@ -0,0 +1,34 @@
+if !System.get_env("EXERCISM_TEST_EXAMPLES") do
+  Code.load_file("flatten_array.exs")
+end
+
+ExUnit.start
+ExUnit.configure exclude: :pending, trace: true
+
+defmodule ChangeTest do
+  use ExUnit.Case
+
+  test "returns original list if there is nothing to flatten" do
+    assert Flattener.flatten([1, 2, 3]) ==  [1, 2, 3]
+  end
+
+  @tag :pending
+  test "flattens an empty nested list" do
+    assert Flattener.flatten([[]]) ==  []
+  end
+
+  @tag :pending
+  test "flattens a not empty nested list" do
```

Comment:
This is probably just "flattens a nested list" as this is kind of the happy path test.